packages/ts-codegen/src/__tests__/index.test.ts

```diff
@@ -1975,6 +1975,55 @@ describe(generateTS, () => {
       "
     `);
   });
+
+  it("legacy no models", () => {
+    const { models, resolvers, enums, inputs } = runGenerateTest(
+      graphql`
+        scalar Foo @model(tsType: "string")
+
+        type Bar @model(tsType: "BarModel", from: "./bar_model.interface") {
+          id: ID!
+        }
+      `,
+      {
+        legacyNoModelsForObjects: true,
+      },
+    );
+
+    expect(enums).toMatchInlineSnapshot(`""`);
+    expect(inputs).toMatchInlineSnapshot(`
+      "import * as Models from "./models.interface";
+      "
+    `);
+    expect(models).toMatchInlineSnapshot(`
+      "import * as Enums from "./enums.interface";
+      export * from "./enums.interface";
+      // Base type for all models. Enables automatic resolution of abstract GraphQL types (interfaces, unions)
+      export interface BaseModel {
+          readonly __typename?: string;
+      }
+      export type Foo = string;
+      export interface Bar extends BaseModel {
+          readonly __typename?: "Bar";
+          readonly id: string;
+      }
+      "
+    `);
+    expect(resolvers).toMatchInlineSnapshot(`
+      "import type { PromiseOrValue } from "@graphitation/supermassive";
+      import type { ResolveInfo } from "@graphitation/supermassive";
+      import * as Models from "./models.interface";
+      import * as Inputs from "./inputs.interface";
+      export * from "./inputs.interface";
+      export declare namespace Bar {
+          export interface Resolvers {
+              readonly id?: id;
+          }
+          export type id = (model: Models.Bar, args: {}, context: unknown, info: ResolveInfo) => PromiseOrValue<string>;
+      }
+      "
+    `);
+  });
```

Comment:
It is not clear here what the expected output change is. Please keep tests lean so it clearly shows what the behaviour is. E.g. assuming that all other output is the same as normal codegen, can all this noise be reduced to something like this?

```ts
expect(models).not.toMatch("the model field/import/etc")
```